test/TestUtilities/Test.Utility/Signing/SigningTestUtility.cs

```diff
@@ -661,6 +661,11 @@ public static void VerifyByteArrays(byte[] expected, byte[] actual)
         }
 
         public static void AssertOfflineRevocation(IEnumerable<ILogMessage> issues, LogLevel logLevel)
+        {
+            AssertOfflineRevocation(issues, logLevel, NuGetLogCode.NU3018);
+        }
+
+        public static void AssertOfflineRevocation(IEnumerable<ILogMessage> issues, LogLevel logLevel, NuGetLogCode code)
```

Comment:
I think these 2 methods can be refactored to a single method because the only difference is the message. May be we can add additional parameter whose value is checked against the `issues` collection (or) invoke [`Assert.All`](https://github.com/xunit/assert.xunit/blob/92395f59e3ff47d6d5a1edeced9e99ccfef9fd37/CollectionAsserts.cs#L24)
```cs
public static void AssertOfflineRevocation
public static void AssertRevocationStatusUnknown

Reply:
Thanks for the review! I disagree about refactoring, since the methods are already fairly minimal on their own. I think refactoring them further would be a bit much, no? The bulk of their bodies is, after all, just picking that message, as you say.